pkg/config/config.go

```diff
@@ -46,6 +47,11 @@ type Config struct {
 	HighwayGRPCEndpoint string
 	HighwayHTTPEndpoint string
 
+	// JWT
+	JWT_Secret        string
```

Comment:
These could be named without underscores ('JWTSecret, JWTSigningMethod, etc) for more consistency with Go standards.